### DISCUSSION TITLE
TuneIn Premium

### DISCUSSION THREAD

**Discussion by chester512** [music-assistant/support#947] [Feature requests and ideas]
Any chance of adding support for TuneIn Premium?

**Comment by OzGav** [comment]
TuneIn works. What in premium do you need?

**Comment by chester512** [comment]
No commercials with tunein premium and I can stream cnn.

**Comment by OzGav** [comment]
Can’t you add the CNN stream to your library?

**Comment by chester512** [comment]
The channel is listed on music assistant via tune in, but says (not
available).

**Comment by OzGav** [comment]
Relates to v1
